features/web-animations.yml

```
@@ -1,5 +1,5 @@
 name: Web animations
-description: The `animate()` method of `Element` objects programmatically animates elements over time and can synchronize the animations of multiple elements.
+description: The `animate()` method, together with the `Animation` interface, animates an element over time, possibly synchronizing the animations of multiple elements. The `getAnimations()` method queries and modifies current CSS animations, transitions, and web animations.
```

Hmm, after reading this it's almost as if we're trying to do too much to tease out the individual pieces of the API. Instead, I'm wondering if we can't give an overview and hint at what things to go look up in the docs. What do you think of something like this?

```suggestion
description: The `animate()` method of `Element` objects, the `document.getAnimations()` method, and the `Animation` interface animate elements over time and coordinate elements' scripted animations, CSS animations, and transitions.
```